components/server/src/api/user.ts

```diff
@@ -74,27 +76,37 @@ export class APIUserService implements ServiceImpl<typeof UserServiceInterface>
 
         // TODO: Once connect-node supports middlewares, lift the tracing into the middleware.
         const trace = {};
-        // TODO for now we use SYSTEM_USER, since it is only called by internal componenets like usage
+        // TODO(gpl) for now we use SYSTEM_USER, since it is only called by internal componenets like usage
         // and not exposed publically, but there should be better way to get an authenticated user
-        await this.userService.blockUser(SYSTEM_USER, userId, true);
-        log.info(`Blocked user ${userId}.`, {
-            userId,
-            reason,
-        });
+        await runWithRequestContext(
```

Comment:
@geropl is there a way to verify it in preview envs?

Reply:
Looking through it, I don't think it has any client, yet :astonished: Thought agent-smith switched long ago, but that's still with json-rpc.

Reply:
So we better remove it? But maybe in a separate PR.